source/_components/media_player.pjlink.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: "PJLink"
+description: "Instructions on how to integrate PJLink enabled projectors into Home Assistant."
+date: 2018-06-21 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Media Player
+ha_release: TBD
+ha_iot_class: "Local Polling"
+---
```

Comment:
Maybe add the PJLink logo?
This would make the component more recognizable and just looks prettier.